docs/share_usage.md

```diff
@@ -0,0 +1,299 @@
+# File Persona usage guide
+
+The HPE 3PAR File Persona feature allows user to manage file 
+shares on 3PAR arrays through Docker interface.
+
+In order to use HPE 3PAR File Persona feature, user needs to 
+configure a backend one for each target array as below:
+
+## Configuring backend for file share
+
+```sh
+[DEFAULT]
+
+# ssh key file required for driver ssh communication with array
+ssh_hosts_key_file = /root/.ssh/known_hosts
+
+# IP Address and port number of the ETCD instance
+# to be used for storing the share meta data
+host_etcd_ip_address =  10.50.164.1
+host_etcd_port_number = 2379
+
+# Client certificate and key details for secured ETCD cluster
+# host_etcd_client_cert = /root/plugin/certs/client.pem
+# host_etcd_client_key = /root/plugin/certs/client-key.pem
+
+# Logging level for the plugin logs
+logging = DEBUG
+
+# Logging level for 3PAR client logs
+hpe3par_debug = True
+
+# Suppress Requests Library SSL warnings
+suppress_requests_ssl_warnings = True
+
+# Set the driver to be File driver
+hpedockerplugin_driver = hpedockerplugin.hpe.hpe_3par_file.HPE3PARFileDriver
+
+hpe3par_api_url = https://10.50.3.24:8080/api/v1
+hpe3par_username = 3paradm
+hpe3par_password = 3pardata
+hpe3par_san_ip = 10.50.3.24
+hpe3par_san_login = 3paradm
+hpe3par_san_password = 3pardata
+
+# Server IP pool is mandatory and can be specified as a mix of range of IPs and
+# individual IPs delimited by comma
+# Each range or individual IP must be followed by the corresponding subnet mask
+# delimited by semi-colon
+# E.g.: IP-Range:Subnet-Mask,Individual-IP:SubnetMask…
+hpe3par_server_ip_pool = 192.168.98.8-192.168.98.13:255.255.192.0
+
+# Default size of FPG to be in the range 1TiB – 64TiB. If not specified here, it defaults to 64
+hpe3par_default_fpg_size = 10 
+```
+User can define multiple backends in case more than one array needs to be managed by the plugin.
+
+User can also define backends for block driver(s) along with file driver(s). 
+However, a default backend is mandatory for both block and file drivers for the default use cases 
+to work. Since ‘DEFAULT’ section can be consumed by either 
+block or file driver but not both at the same time, the other driver
+is left out without a default backend. In order to satisfy the need for the other 
+driver to have default backend, HPE 3PAR Plugin introduces two new keywords to 
+denote default backend names to be used in such a situation:
+1. DEFAULT_FILE and
+2. DEFAULT_BLOCK
+
+In case where user already has ‘DEFAULT’ backend configured for 
+block driver, and file driver also needs to be configured, then 
+‘DEFAULT_FILE’ backend MUST be defined. In this case, if there 
+is a non-default backend defined for file driver without 
+'DEFAULT_FILE' backend defined, plugin won't get initialized 
```

Comment:
This needs modification at this part it is bit confusing make sure we capture all these conditions at know limitation section.

Reply:
This is not a limitation @prablr79 , rather it is by design which the user needs to be aware of. I've added an example to make the explanation more clear.

Please have a look at the updated content at:
[share_usage.md](https://github.com/imran-ansari/python-hpedockerplugin/blob/dev/docs/share_usage.md)